samples/winapi_filefind/main.c

```diff
@@ -44,17 +49,31 @@ int main(void)
 
     debugPrint("\n");
 
-    DWORD error = GetLastError();
+    error = GetLastError();
     if (error == ERROR_NO_MORE_FILES) {
-        debugPrint("Done!\n");
+        debugPrint("Done! Re-running in 10 seconds...\n");
     } else {
+        errored = true;
         debugPrint("error: %x\n", error);
     }
 
     FindClose(hFind);
 
-    while (1) {
-        Sleep(2000);
+    // Give some time to the user to read the files and directories
+    Sleep(10000);
+
+    ret = nxUnmountDrive('C');
+    // If there was an error while unmounting (nxUnmountDrive() returns false)
```

Comment:
This claims that it will ~"return false". This isn't actually happening anywhere.